engine/src/main/java/org/terasology/rendering/dag/nodes/AmbientOcclusionNode.java

```diff
@@ -114,7 +104,7 @@ public void process() {
 
     @Override
     public void update() {
-        ssaoFbo = displayResolutionDependentFBOs.get(SSAO_FBO);
+        FBO ssaoFbo = displayResolutionDependentFBOs.get(SSAO_FBO);
```

Comment:
We discussed this already, but please remind me: update() is executed, for example, when the user resize the window. Why not to avoid any form of memory allocation in those circumstances, even if it's admittedly small in this case?